_topic_maps/_topic_map.yml

```diff
@@ -1180,6 +1180,8 @@ Topics:
     File: persistent-storage-csi-azure-stack-hub
   - Name: GCP PD CSI Driver Operator
     File: persistent-storage-csi-gcp-pd
+  - Name: IBM Cloud CSI Driver Operator
```

Comment:
@dobsonj What about name update to "IBM VPC Block CSI Driver Operator"

Reply:
> @dobsonj What about name update to "IBM VPC Block CSI Driver Operator"

You're right, that would be more clear, especially if they introduce other storage drivers in the future.